docs/operations/downgrade-ondat-2.7-to-2.6.md

```diff
@@ -0,0 +1,54 @@
+---
```

Comment:
I'm sure you have, but has anyone else run through the procedure and checked it works? I can try and do it today if I get time later.

Reply:
@DavidMarchant has :D lol

Feel free to test this out however

Reply:
Yes so I have tested it & I am confident in the procedure and the quality of the script

That said, we & customer success are in agreement that this is something that we want to have a hand in customers doing which is why we have the disclaimer about customers opening a support case before going through with it

We have an assurance for all combinations of orchestrators/cluster configs/ctl tool versions etc.